db/migrations/0004_alter_order_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-11-18 11:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_order_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(blank=True, null=True),

```

Comment:
The `created_at` field is now set to allow `null` and `blank` values. This change allows for orders to exist without a `created_at` timestamp. Ensure that this aligns with your application's requirements and that your logic can handle orders with missing `created_at` values.